src/modules/site-v2/base/static/js/plot.js

```diff
@@ -54,7 +54,15 @@ function add_histogram_along_axis(d, axis, data, target, config={}) {
   const position     = config['position'] || [0, 0];
   const graph_height = config['height']   || 60;
   const color        = config['color']    || 'black';
-  const num_bins     = config['num_bins'] || ((axis.ticks().length - 1) * (config['bins_per_tick'] || 1));
+
+  // Compute the approximate number of bins there should be
+  let num_bins;
+  if (config['bins_per_tick']) {
+    num_bins = (axis.ticks().length - 1) * config['bins_per_tick']
+  } else {
+    const bin_width = config['bin_width'] || ((axis.ticks().length - 1) * 2)
```

Comment:
do we worry about the scenario where `config['bin_width'] === 0` is what is intended? In the case, 0 || N will yield to N.